src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -38,17 +32,16 @@
             + "Existing values will be overwritten by the input values.\n"
             + "Parameters: INDEX (must be a positive integer) "
             + "[" + PREFIX_NAME + "NAME] "
-            + "[" + PREFIX_PHONE + "PHONE] "
-            + "[" + PREFIX_EMAIL + "EMAIL] "
-            + "[" + PREFIX_ADDRESS + "ADDRESS] "
-            + "[" + PREFIX_TAG + "TAG]...\n"
+            + "[" + PREFIX_STUDENTID + "STUDENTID] "
+            + "[" + PREFIX_CLASSID + "CLASSID] "
+            + "[" + PREFIX_GRADE + "GRADE] "
             + "Example: " + COMMAND_WORD + " 1 "
-            + PREFIX_PHONE + "91234567 "
-            + PREFIX_EMAIL + "johndoe@example.com";
+            + PREFIX_STUDENTID + "A0121234H "
+            + PREFIX_CLASSID + "B01";
 
     public static final String MESSAGE_EDIT_PERSON_SUCCESS = "Edited Person: %1$s";
     public static final String MESSAGE_NOT_EDITED = "At least one field to edit must be provided.";
-    public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the address book.";
+    public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the ProgrammerError.";
```

Comment:
Should we omit the 'the'? i.e. "This persona already exists in ProgrammerError."

Reply:
I think we can!

Reply:
I think allard suggestion is good!